internal/resource/service_test.go

```diff
@@ -52,6 +53,8 @@ func TestService_ProcessDefaults(t *testing.T) {
 		r.Service.Retries = 1
 		r.Service.Protocol = "grpc"
 		r.Service.Enabled = wrapperspb.Bool(false)
+		tlsVerify := false
+		r.Service.TlsVerify = &tlsVerify
```

Comment:
Just fyi: you could already use `r.Service.TlsVerify = falsy`

Reply:
interesting, getting `undefined: falsy` error here?